docs/developer/pandas/partition_api.rst

```diff
@@ -0,0 +1,79 @@
+Pandas Partition API in Modin
+=============================
+
+If you are working with a Modin Dataframe, you can unwrap its remote partitions
+to get the raw future objects compatible with the execution engine (e.g. ``ray.ObjectRef`` for Ray).
+You can use this API to get the IPs of the nodes that hold these objects as well. In that case you can pass the partitions
+having needed IPs to your function. It can help with minimazing of data movement between nodes. In addition to
+unwrapping of the remote partitions we also provide API to construct Modin DataFrame from them.
+
+Ray engine
+----------
+However, it is worth noting that for Modin on ``Ray`` engine with ``pandas`` backend IPs of the remote partitions may not match
+actual locations if the partitions are lower than 100 kB. Ray saves such objects (<= 100 kB, by default) in in-process store
+of the calling process (please, refer to `Ray documentation`_ for more information). We can't get IPs for such objects while maintaining good performance.
+So, you should keep in mind this for unwrapping of the remote partitions with their IPs. Several options are provided to handle the case in
+``How to handle Ray objects that are lower 100 kB`` section.
+
+Dask engine
+-----------
+There is no mentioned above issue for Modin on ``Dask`` engine with ``pandas`` backend because ``Dask`` saves any objects
+in the worker process that processes a function (please, refer to `Dask documentation`_ for more information).
+
+Install Modin Pandas Partition API
```

Comment:
I don't think this installation section is providing any value.

Reply:
I agree, removed.